lib/internal/http2/compat.js

```diff
@@ -89,6 +92,17 @@ function statusMessageWarn() {
   }
 }
 
+function connectionHeaderMessageWarn() {
+  if (statusConnectionHeaderWarned === false) {
+    process.emitWarning(
+      `The provided connection header is not valid, ` +
+      'it is supported only for compatibility.',
```

Comment:
So the warning says it is supported, but it is just dropped. I would really call this "supported". Can the message be more clear that the value you set (and thus the client behavior you would except from the client) is just dropped and never given to the client?

Reply:
Sure 👍

Reply:
What about: 
"The provided connection header is not valid, the value will dropped from the header and will never be in use."

Reply:
Sgtm

Reply:
👍